iot_input/data/ir_cron_data.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="ir_cron_mail_notify_channel_moderators" model="ir.cron">
+            <field name="name">IOT: Clear iot.device.input.action table</field>
+            <field name="model_id" ref="model_iot_device_input_action"/>
+            <field name="state">code</field>
+            <field name="code">model.search([]).unlink()</field>
```

Comment:
That is true but I can't bring my head around to think: Is this possible in a cron action?

Reply:
Yes, it is possible, but it would be better to add this on a internal funcionat on iot.device.input.action

Similar to:

https://github.com/OCA/queue/blob/14.0/queue_job/models/queue_job.py#L253